### PR TITLE
Fix phpdoc

### PR DESCRIPTION
**Pull request by VincentLanglet**
When running phpstan, I discover that an error was coming from a wrong phpDoc in these files.

### CODE REVIEW

File: lib/Doctrine/ORM/Query.php [hunk 1]
```diff
@@ -624,7 +624,7 @@ public function contains($dql)
     /**
      * Sets the position of the first result to retrieve (the "offset").
      *
-     * @param integer $firstResult The first result to return.
+     * @param integer|null $firstResult The first result to return.
```
Comment (greg0ire) [inline lib/Doctrine/ORM/Query.php]:
```suggestion
     * @param int|null $firstResult The first result to return.
```

File: lib/Doctrine/ORM/QueryBuilder.php [hunk 2]
```diff
@@ -100,7 +100,7 @@ class QueryBuilder
     /**
      * The index of the first result to retrieve.
      *
-     * @var integer
+     * @var integer|null
```
Comment (greg0ire) [inline lib/Doctrine/ORM/QueryBuilder.php]:
```suggestion
     * @var int|null
```

File: lib/Doctrine/ORM/QueryBuilder.php [hunk 3]
```diff
@@ -616,7 +616,7 @@ function (Query\Parameter $parameter) use ($key) : bool {
     /**
      * Sets the position of the first result to retrieve (the "offset").
      *
-     * @param integer $firstResult The first result to return.
+     * @param integer|null $firstResult The first result to return.
```
Comment (greg0ire) [inline lib/Doctrine/ORM/QueryBuilder.php]:
```suggestion
     * @param int|null $firstResult The first result to return.
```

File: lib/Doctrine/ORM/QueryBuilder.php [hunk 4]
```diff
@@ -631,7 +631,7 @@ public function setFirstResult($firstResult)
      * Gets the position of the first result the query object was set to retrieve (the "offset").
      * Returns NULL if {@link setFirstResult} was not applied to this QueryBuilder.
      *
-     * @return integer The position of the first result.
+     * @return integer|null The position of the first result.
```
Comment (greg0ire) [inline lib/Doctrine/ORM/QueryBuilder.php]:
```suggestion
     * @return int|null The position of the first result.
```

**Review by greg0ire — COMMENTED** [review]
In case you are wondering what's up with phpcs, we have so many issues with cs on this project that checks are only enabled on PR diffs. That's why `integer` was ok before but isn't now.

**Comment by VincentLanglet** [comment]
@greg0ire Thanks

**Review by greg0ire — APPROVED** [review]

**Comment by greg0ire** [comment]
Thanks @VincentLanglet !

**Pull request merged**